frontend/package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^9.4.0",
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.19.2",
+    "bootstrap-icons": "^1.0.0-alpha2",
     "node-sass": "^4.13.1",
```

Comment:
Do we have to modify anything in our setup with the addition of this line?

Reply:
Just that when this is merged to master, you guys will also have to install this dependency before running the `npm start` command, I believe.